src/FileAdder/FileAdder.php

```diff
@@ -290,7 +290,11 @@ public function toMediaCollection(string $collectionName = 'default', string $di
 
         $media->fill($this->properties);
 
-        $this->attachMedia($media);
+        $newMedia = $this->attachMedia($media);
+
+        if(isset($this->getMediaCollection($collectionName)->singleFile) && $this->getMediaCollection($collectionName)->singleFile) {
```

Comment:
That `isset` check isn't necessary because the default is `false` right?

Reply:
The other tests and tests for other collections break since there is no property called `singleFile` defined.